src/__constants__/statuses.js

```diff
@@ -0,0 +1,9 @@
+const STATUSES = {
+  BOOKED: 'BOOKED',
+  PENDING: 'PENDING',
+  CANCELED: 'CANCELED',
+  FUTURE: 'Future',
```

Comment:
Why `CANCELED` status equals to `'CANCELED'`, but `FUTURE` - to `'Future'`?! Use same style (uppercase)